.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: c
 env:
-  - OCAML_VERSION=4.08.0
+  - CI_KIND=build OCAML_VERSION=4.08.0
+  - CI_KIND=changes OCAML_VERSION=4.08.0
```

Comment:
it looks like it is running twice the same build and not checking the changelog at all

Reply:
Why would `CI_KIND` have any effect ? That's just passed to our scripts, we have to interpret it.